src/methodology-modal/MethodologyModal.js

```diff
@@ -21,40 +23,503 @@ const SectionTitle = styled(HeadingTitle)`
   font-size: 20px;
 `;
 
+const CONTENTS_BY_PAGE = {
+  [PATHS.overview]: null,
+  [PATHS.sentencing]: (
+    <>
+      <SectionTitle>{ALL_PAGES.get(PATHS.sentencing)}</SectionTitle>
+      <MethodologyDescription>
+        This page includes information about everyone currently involved with
+        the ND DOCR.
+      </MethodologyDescription>
+      <MethodologyDescription>
+        District selection allows you to filter data in a section to only
+        individuals whose sentence was imposed by a judge from a particular
+        judicial district. Individuals are linked to the judicial district
+        associated with their charge (for individuals sentenced to
+        incarceration) or to the judicial district that encompasses the county
+        in which their offense took place (for individuals sentenced to
+        probation).
+      </MethodologyDescription>
+      <MethodologyDescription>
+        Some individuals have multiple sentences resulting from multiple
+        charges. When this occurs, one charge is considered the “controlling”
+        charge, meaning the sentence associated with this charge is currently
+        the most restrictive sentence being served. For individuals currently
+        incarcerated, the sentence associated with the controlling charge
+        determines the judicial district in which the person is counted.
+        Individuals on supervision are counted in each judicial district for
+        which they are serving a sentence.
+      </MethodologyDescription>
+      <MethodologyDescription>
+        When distribution of sentences by age are shown, individuals are counted
+        towards the age group they fall into as of today. Gender distributions
+        only include male and female due to low numbers of other reported
+        genders. Individuals may have more than one reported race or ethnicity.
+        For the purpose of this dashboard, distributions by race count
+        individuals with more than one reported race or ethnicity towards the
+        racial or ethnic identity that is least represented in North Dakota’s
+        population. The “Other” category includes Asian and Native Hawaiian or
+        other Pacific Islander individuals due to low counts of these reported
```

Comment:
Since we capitalized "Other" on the Racial Disparities page, let's do the same everywhere we state this race